src/branch_view.js

```diff
@@ -225,6 +225,9 @@ branch_view._getPageviewRequestData = function(metadata, options, branch, isDism
 	if (linkClickId) {
 		obj.data['link_click_id'] = linkClickId;
 	}
+	if (sessionStorage['data'] && JSON.parse(sessionStorage['data'])['+referrer']) {
+		obj.data['+referrer'] = JSON.parse(sessionStorage['data'])['+referrer'];
+	}
```

Comment:
we are very strict about what we send out to the API, so I need to add it to this request thing

Reply:
Can we make this simpler since we're doing JSON.parse() twice?